src/Agent.Plugins/PipelineCache/TarUtils.cs

```diff
@@ -74,7 +74,8 @@ public static class TarUtils
 
             Directory.CreateDirectory(targetDirectory);
             
-            DedupIdentifier dedupId = DedupIdentifier.Create(manifest.Items.Single(i => i.Path.EndsWith(archive, StringComparison.OrdinalIgnoreCase)).Blob.Id);
+            DedupIdentifier dedupId = DedupIdentifier.Create(manifest.Items.Single(i => i.Path.EndsWith(archive, StringComparison.OrdinalIgnoreCase)).Blob.Id);          
+
```

Comment:
Again... sorry if this is out of place :P but this line has trailing spaces hehe